src/main/java/com/tutrit/bean/Customer.java

```diff
@@ -0,0 +1,8 @@
+package com.tutrit.bean;
+
+public record Customer(String id
+        , String name
```

Comment:
comma on a wrong line. Doesn't checkstyle should comply on that?

Reply:
I solved this.
we dosen't have checkstyle in this progect